### DISCUSSION TITLE
Is it possible to use the `profile.html`of `homepage.layout` in the other menu?

### DISCUSSION THREAD

**Discussion by sevi418** [jpanther/congo#8] [Q&A] [answered]
Is it possible to use the `profile.html` of`homepage.layout` in the other menu?

like this menu, I would like to put my profile in the "about" menu, not the homepage, what should I do?
![](https://raw.githubusercontent.com/SevicheCen/obsidian-image/main/img202109281847695.png)

I have try to set the menu as:
`[[main]]`
`  name = "About"`
  `pageRef= "partials/home/profile.html" `
  `weight = 40`

But it doesn't work, so what is the right way to do that?

I know it is a stupid question. if you know how to solve the problem, could you help me?

**Comment by jpanther** [comment, marked as answer]
Thanks for the question. While it's not possible to link directly to a partial in Hugo, you can create your own layout for an about page that will include the profile. There's a couple of ways you could achieve this but you'll need to do something like the following:

1. Make an about page in your content folder: `content/about/_index.md`
```yaml
---
title: About
---
```
2. In your layouts folder (which you may need to create), create a custom layout for the about page that includes the profile partial: `layouts/about/list.html`
```go
{{ define "main" }}
  {{ partial "partials/home/profile.html" . }}
{{ end }}
```
3. Update your menu to `pageRef = "about"`
4. Rebuild the Hugo site

**Reply by sevi418** [reply]
Wow, It works! Thank you Jpanther

**Reply by jpanther** [reply]
Excellent, glad it helped!

**Reply by dmarks100** [reply]
Extremely helpful, thanks!
